### DISCUSSION TITLE
How to configure DASHBOARD_RBAC

### DISCUSSION THREAD

**Discussion by srinisubramanian** [apache/superset#19286] [Q&A / Help]
I am at my wits end to get this working.  Have enabled DASHBOARD_RBAC and then assigned dashboards to roles.  Irrespective of what I do the dashboards dont show up.  in fact even dashboards where there are no roles assigned but the user has access to their datasets also dont show up.  Other than admin, no other user sees dashboard.  I have tried by giving access to existing dashboards and created new ones.  No difference.

How exactly is this to be configured? Using Superset 1.4.1

**Comment by srinisubramanian** [comment]
@srinify any info or doc links?

**Reply by sossaman** [reply]
Hello,
I find myself in the same situation. The documentation is unclear in this regard.
